libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/utils/scheduledMailRecipients.ts

```diff
@@ -10,3 +16,51 @@ export const getScheduledEmailRecipientEmail = (recipient: IScheduleEmailRecipie
 
 export const getScheduledEmailRecipientDisplayName = (recipient: IScheduleEmailRecipient): string =>
     isScheduleEmailExistingRecipient(recipient) ? userFullName(recipient.user)! : recipient.email;
+
+const scheduleEmailRecipientDelimiter = /[,;\s]/;
+
+export const splitScheduledEmailRecipients = (
+    recipients: IScheduleEmailRecipient[],
+): IScheduleEmailRecipient[] => {
+    return flatMap(recipients, (recipient) => {
+        return splitScheduledEmailRecipientByDelimiter(recipient, scheduleEmailRecipientDelimiter);
+    });
+};
+
+export const uniqueScheduledEmailRecipients = (
+    recipients: IScheduleEmailRecipient[],
+): IScheduleEmailRecipient[] => {
+    const recipientIds: string[] = [];
+    return recipients.filter((recipient) => {
+        const recipientId = getScheduledEmailRecipientUniqueIdentifier(recipient);
+        if (recipientIds.includes(recipientId)) {
+            return false;
+        } else {
+            recipientIds.push(recipientId);
+            return true;
+        }
+    });
+};
+
+const splitScheduledEmailRecipientByDelimiter = (
+    recipient: IScheduleEmailRecipient,
+    delimiter: RegExp,
+): IScheduleEmailRecipient[] => {
+    if (isScheduleEmailExistingRecipient(recipient)) {
+        return [recipient];
+    }
+    return splitScheduledEmailExternalRecipientByDelimiter(recipient, delimiter);
+};
+
+const splitScheduledEmailExternalRecipientByDelimiter = (
+    recipient: IScheduleEmailExternalRecipient,
+    delimiter: RegExp,
+): IScheduleEmailExternalRecipient[] => {
+    return recipient.email
+        .split(delimiter)
+        .map((val) => val.trim())
+        .filter((val) => !isEmpty(val))
+        .map((val) => {
+            return { email: val };
+        });
```

Comment:
This is just minor improvement but if you would rename the parameter of the map function, this could be a one liner, i.e., `.map((email) => ({ email }))`. However, you can leave it as it is. There is nothing wrong with it :)